Makefile

```diff
@@ -219,7 +219,13 @@ dev-dracon: deploy-elasticoperator deploy-arangodb-crds add-bitnami-repo
 		  --values ./deploy/dracon/values/dev.yaml \
 		  --create-namespace \
 		  --namespace $(DRACON_NS) \
-		  --set "deduplication-db-migrations.image.tag=$(DRACON_VERSION)"
+		  --wait
+	@helm upgrade deduplication-db-migrations ./deploy/deduplication-db-migrations/chart \
```

Comment:
this now depends on the draconctl image, which doesn't get automatically published locally, and as such the quickstart guide on a fresh install doesn't work anymore

i.e. running
`make dev-deploy DRACON_VERSION=v0.19.0` on a fresh install crashes with the `draconctl` image not being found